frontend/src/app/core/services/user.service.ts

```diff
@@ -61,7 +63,7 @@ export class UserService {
     return this.httpClient.put<void>(`${this.registrationPageWebAPIUrl}/user`, updateUserDTO, httpOptions);
   }
 
-  loginUser(userForLogin: UserForLoginDTO): Observable<string> {
-    return this.httpClient.post<string>(`${this.registrationPageWebAPIUrl}/login`, userForLogin);
+  loginUser(userForLogin: UserForLoginDTO): Observable<AuthenticationResponseDTO> {
```

Comment:
if it is changed here does it work with be? When changing the be you can also change the way auth works. So Bearer gets added on fe not be

Reply:
This works with BE. Before we used to get a string and then cast it to AuthenticationResponseDTO which didn't make much sense so i changed it